h/security/policy/top_level.py

```diff
@@ -34,9 +35,23 @@ def _load_identity(self, request):
 @RequestLocalCache()
 def get_subpolicy(request):
     """Return the subpolicy for TopLevelSecurityPolicy to delegate to for `request`."""
+    cookie = webob.cookies.SignedCookieProfile(
+        secret=request.registry.settings["h_auth_cookie_secret"],
+        salt="authsanity",
+        cookie_name="auth",
+        max_age=30 * 24 * 3600,  # 30 days
+        httponly=True,
+        secure=request.scheme == "https",
+    )
+    cookie = cookie.bind(request)
+
     if is_api_request(request):
         return APIPolicy(
-            [BearerTokenPolicy(), AuthClientPolicy(), APICookiePolicy(CookiePolicy())]
+            [
+                BearerTokenPolicy(),
+                AuthClientPolicy(),
+                APICookiePolicy(CookiePolicy(cookie)),
+            ]
         )
 
-    return CookiePolicy()
+    return CookiePolicy(cookie)
```

Comment:
This passes the same cookie profile to `CookiePolicy()` as to `APICookiePolicy()`, maintaining the current single-cookie behaviour for now. But a simple change to this `get_subpolicy()` function could now create two separate `SignedCookieProfile`'s (e.g. one with `samesite="lax"` and one with `samesite="strict"`) and pass one profile to `CookiePolicy()` and the other to `APICookiePolicy()`.